homeassistant/components/seventeentrack/services.py

```diff
@@ -89,7 +89,9 @@ async def get_packages(call: ServiceCall) -> ServiceResponse:
                     ATTR_TRACKING_NUMBER: package.tracking_number,
                     ATTR_LOCATION: package.location,
                     ATTR_STATUS: package.status,
-                    ATTR_TIMESTAMP: package.timestamp.isoformat(),
+                    ATTR_TIMESTAMP: package.timestamp.isoformat()
+                    if package.timestamp
+                    else "",
```

Comment:
```suggestion
                    ATTR_TIMESTAMP: (
                        package.timestamp.isoformat() if package.timestamp else ""
                    ),
```

Reply:
I missed the comment from @dgomes above.
Please take it into account to decide if it should return None, empty string, or skip the attribute altogether.

In which case it might be easier to have a function to convert a package into attributes:
```python
        return {
            "packages": [
                package_to_dict(package)
                for package in live_packages
                if slugify(package.status) in package_states or package_states == []
            ]
        }
```